config/config.go

```diff
@@ -117,6 +118,8 @@ type Config struct {
 	AccessLogStripQuery                 bool      `yaml:"access-log-strip-query"`
 	SuppressRouteUpdateLogs             bool      `yaml:"suppress-route-update-logs"`
 
+	OpenTelemetry *otel.Options `yaml:"open-telemetry"`
```

Comment:
Maybe better to tell it's OTelOptions or so

Reply:
I'd like it to match the flag name which looks ok. Let me think about it and I'll change it later if needed.